php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -445,16 +441,13 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                 }
                 switch($type) {
                 case 'page':
-                    $parsingPage = trim($pieces[2]);
-                    $pageName    = $db->pselectOne(
-                        "SELECT Subtest_name
-                        FROM instrument_subtests
-                        WHERE Test_name=:testinst AND Description=:parsing",
-                        [
-                            'testinst' => $this->testName,
-                            'parsing'  => $parsingPage,
-                        ]
-                    );
+                    $pageDescription  = trim($pieces[2]);
+                    $pageName         = "page_$subtestCount";
```

Comment:
subtestCount doesn't seem to ever be incremented.. I think this will make every page "page_1"

Reply:
yeah working on another commit that also fixes a bunch of notices

Reply:
done, tested and its working perfectly on my end
see BMI changes